client/src/app/user.service.ts

```diff
@@ -162,7 +163,7 @@ export class UserService {
   /**
    *  Fetches the user's data and stores it into this.userData.
    */
-  getUserData(): Boolean {
+  fetchUserData(): Boolean {
```

Comment:
Not sure, but might need to be a lowercase boolean return type..?

Reply:
Changed to lowercase 'boolean'.